### DISCUSSION TITLE
Multi Language

### DISCUSSION THREAD

**Discussion by MattCW9** [speech-translator-ext/speech-translator-readme#24] [Q&A]
Hello, Is there a way to either run 2 instances of this app, or simultaneously translate speech into two different languages? When I try to run more than one instance, it auto closes my previous window.

**Comment by kappaflow** [comment]
Hello,
Yes it's possible, you can use chrome web browser for one language and edge for another one. Also, you should check the instruction for an interview setup to learn how to use the same browser.
Basically you can use 2 extensions, one for each language.
